coldfront/core/project/models.py

```diff
@@ -15,6 +15,7 @@
 from coldfront.core.utils.common import import_from_settings
 
 PROJECT_ENABLE_PROJECT_REVIEW = import_from_settings('PROJECT_ENABLE_PROJECT_REVIEW', False)
+INSTITUTION_CODE = import_from_settings('INSTITUTION_CODE', False)
```

Comment:
Where is this initialized in the config? I'm not seeing where `INSTITUTION_CODE` is being used. Did you mean `PROJECT_CODE` here?